hooks/conan-center.py

```diff
@@ -667,6 +668,25 @@ def test(out):
                     out.error("The file '{}' uses CRLF. Please, replace by LF."
                               .format(filename))
 
+    @run_test("KB-H063", output)
+    def test(out):
+        def validate_cmake_cxx_std(cmakefile_path):
+            cmake_content = tools.load(cmakefile_path)
+            cxx_std = re.search(r"cxx_std_\w+", cmake_content)
+            if cxx_std:
+                match = re.search(r"cmake_minimum_required\(version [\"']?(\w\.\w+)", cmake_content.lower())
```

Comment:
I would have expected digits not words 🤔 

```suggestion
            cxx_std = re.search(r"cxx_std_\d+", cmake_content)
            if cxx_std:
                match = re.search(r"cmake_minimum_required\(version [\"']?(\d\.\d+)", cmake_content.lower())
```

Reply:
Word is equivalent to [A-Za-z0-9_], but yeah, digits should fit better.